__tests__/command_helpers/checkCLI.ts

```diff
@@ -24,6 +29,10 @@ test('fine on existing binary', async () => {
   expect(await checkCLI(alwaysExistCLI, context)).toBe(undefined)
 })
 
+test('errors with message when an improper semver is sent', async () => {
+  expect(await checkCLI(badSemver, context)).toBe(`Invalid semver rule ${badSemver.semver}`)
+})
+
```

Comment:
I identified a missing test, and added it as lagniappe in this PR.  Sorry for the distraction, but I'm always trying to up the % coverage.